variables.tf

```diff
@@ -9,6 +9,39 @@ variable "database" {
   description = "The name of the first database to be created when the cluster is created"
 }
 
+variable "engine" {
+  type        = string
+  default     = "aurora"
+  description = "The engine type of the Aurora Cluster"
+}
+
+variable "engine_version" {
+  type        = string
+  default     = "5.6.10a"
+  description = "The engine version of the Aurora Cluster, default it is using mysql 5.6"
```

Comment:
```suggestion
  description = "The engine version of the Aurora cluster"
```

Reply:
I wonder if we should infer this automatically based of the selected engine (as this can only be one specific version for MySQL and also only one specific version for PostgreSQL)?